lib/timeline/Timeline.js

```diff
@@ -160,22 +161,21 @@ function Timeline (container, items, groups, options) {
   }
 
   //Single time autoscale/fit
-  this.fitDone = false;
+  this.initialFitDone = false;
   this.on('changed', function (){
     if (this.itemsData == null || this.options.rollingMode) return;
-    if (!me.fitDone) {
-      me.fitDone = true;
+    if (!me.initialFitDone) {
+      me.initialFitDone = true;
       if (me.options.start != undefined || me.options.end != undefined) {
         if (me.options.start == undefined || me.options.end == undefined) {
           var range = me.getItemRange();
         }
 
         var start = me.options.start != undefined ? me.options.start : range.min;
         var end   = me.options.end   != undefined ? me.options.end   : range.max;
-        me.setWindow(start, end, {animation: false});
-      }
-      else {
-        me.fit({animation: false});
+        me.setWindow(start, end, {animation: false}, this._activateOnInitialDrawDone.bind(this));
```

Comment:
Strictly speaking, calling it here is a lie; `_redraw()` is done at the end of this method. `initialDrawDone` should be set to true *after* that call, same applies to the callback.